apps/charterafrica/src/lib/payload/index.js

```diff
@@ -21,17 +21,37 @@ async function getCollection(collection, options) {
   });
 }
 
+async function createCollection(collection, data, options) {
+  return payload.create({
+    collection,
+    data,
+    ...options,
+  });
+}
+
 async function findGlobal(slug, options) {
   return payload.findGlobal({
     ...options,
     slug,
   });
 }
 
+async function updateCollection(collection, data, options) {
```

Comment:
I'm assuming id is required so
```suggestion
async function updateCollection(collection, id, data, options) {
```